### PR TITLE
Collection page stub

### PR DESCRIPTION
**Pull request by davodesign84**

### CODE REVIEW

File: packages/kauri-web/components/containers/Navbar/View.js [hunk 1]
```diff
@@ -91,10 +93,12 @@ class Logo extends React.Component {
 
 export default class Navbar extends React.Component {
   render () {
-    const { userId, routeChangeAction, user, url, confirmationPage } = this.props
+    const { userId, routeChangeAction, user, url, confirmationPage, navcolor } = this.props;
+
+    console.log(navcolor);
```
Comment (rej156) [inline packages/kauri-web/components/containers/Navbar/View.js]: Omit

File: packages/kauri-components/components/Section/CollectionSection.re [hunk 2]
```diff
@@ -0,0 +1,112 @@
...
+  description: string,
+  articles: Js.Nullable.t(array(article)),
+  routeChangeAction: string => unit,
+};
+
+let default =
+  ReasonReact.wrapReasonForJs(
+    ~component,
+    jsProps => {
+      Js.log(jsProps);
```
Comment (rej156) [inline packages/kauri-components/components/Section/CollectionSection.re]: Omit

File: packages/kauri-components/components/Card/ArticleCard.re [hunk 3]
```diff
@@ -116,35 +127,28 @@ let make =
...
+              switch (profileImage) {
+              | Some(image) => image
+              | None => "https://cdn1.vectorstock.com/i/1000x1000/77/15/seamless-polygonal-pattern-vector-13877715.jpg"
+              }
+            )
+          />
+        </div>
       </div>
     </BaseCard>,
+  /* <CardCounter
```
Comment (rej156) [inline packages/kauri-components/components/Card/ArticleCard.re]: Omit maybe

File: packages/kauri-web/components/connections/home-page/HomePageCollections.re [hunk 4]
```diff
@@ -54,8 +54,8 @@ module Styles = {
 
 module GetCollections = [%graphql
```
Comment (rej156) [inline packages/kauri-web/components/connections/home-page/HomePageCollections.re]: Move into own module file inside /queries/Collections_Queries.re

File: packages/kauri-web/routes.js [hunk 5]
```diff
@@ -24,3 +24,4 @@ routes
   .add('article-published', '/article/:article_id/article-version/:article_version/article-published')
   .add('reject-article', '/article/:article_id/article-version/:article_version/reject-article')
   .add('topic-home', '/topic/:category')
+  .add('collection', '/collection/:id')
```
Comment (rej156) [inline packages/kauri-web/routes.js]: Please change to :collection_id

File: packages/kauri-web/components/containers/Collection/View.js [hunk 6]
```diff
@@ -0,0 +1,64 @@
...
+  padding-bottom: 50px;
+  box-shadow: inset 0px 0px 140px 120px rgba(0, 0, 0, 0.5);
+`
+
+const OpaqueContainer = styled(ContentContainer)`
+`;
+
+class CollectionPage extends Component<Props> {
+  render () {
+    const { name, description, date_created, owner, sections } = this.props.data.collection
```
Comment (rej156) [inline packages/kauri-web/components/containers/Collection/View.js]: This.props.data.collection can be null potentially even if we use the withLoading HoC
Might wanna put some defensive coding here

File: packages/kauri-web/components/containers/Collection/View.js [hunk 7]
```diff
@@ -0,0 +1,64 @@
...
+const HeaderContainer = styled(ContentContainer)`
+  background: url('https://source.unsplash.com/collection/1242150') center center;
+  background-size: cover;
+  margin-top: -76px;
+  padding-top: 106px;
+  padding-bottom: 50px;
+  box-shadow: inset 0px 0px 140px 120px rgba(0, 0, 0, 0.5);
+`
+
+const OpaqueContainer = styled(ContentContainer)`
```
Comment (rej156) [inline packages/kauri-web/components/containers/Collection/View.js]: Better to alias or leave with .extend``

File: packages/kauri-components/components/Section/CollectionSection.re [hunk 8]
```diff
@@ -0,0 +1,112 @@
...
+};
+
+let component = ReasonReact.statelessComponent("CollectionSection");
+
+let make = (~routeChangeAction, ~name, ~description="", ~articles, _children) => {
+  ...component,
+  render: _self =>
+    switch (articles) {
+    | None => ReasonReact.null
+    | Some(x) =>
```
Comment (rej156) [inline packages/kauri-components/components/Section/CollectionSection.re]: Might wanna rename the x var in the future

File: packages/kauri-components/components/Section/CollectionSection.re [hunk 9]
```diff
@@ -0,0 +1,112 @@
...
+  user,
+  profileImage: string,
+  article_version: int,
+};
+
+module Styles = {
+  let container =
+    Css.(
+      [%css
+        {|
```
Comment (rej156) [inline packages/kauri-components/components/Section/CollectionSection.re]: Note to selves to convert to not using the PPX

File: packages/kauri-components/components/Headers/CollectionHeader.re [hunk 10]
```diff
@@ -0,0 +1,56 @@
...
+      ~name,
+      ~description,
+      ~username,
+      ~updated,
+      _children,
+    ) => {
+  ...component,
+  render: _self =>
+    <div className=Styles.container>
+      <Heading size=28 text=name color="ffffff" />
```
Comment (rej156) [inline packages/kauri-components/components/Headers/CollectionHeader.re]: We might want to default this color hex as well as using variant types eventually

File: packages/kauri-web/components/containers/Navbar/View.js [hunk 11]
```diff
@@ -15,7 +15,8 @@ const StyledMenu = styled(Menu)`
   display: flex;
   height: ${menuHeaderHeight}px !important;
   line-height: ${menuHeaderHeight}px !important;
-  background-color: ${props => props.confirmationPage && props.theme.secondaryColor};
+  background-color: ${props => props.navcolor ? props.navcolor : props.confirmationPage && props.theme.secondaryColor};
```
Comment (rej156) [inline packages/kauri-web/components/containers/Navbar/View.js]: Maybe refactor to use props.type === 'confirmationPage' 
Ternaries and boolean props for boolean conditionals gets messy

File: packages/kauri-web/queries/Collection.js [hunk 12]
```diff
@@ -0,0 +1,35 @@
+import gql from 'graphql-tag'
+
+export const globalCollectionDetails = gql`
```
Comment (rej156) [inline packages/kauri-web/queries/Collection.js]: Use ReasonApollo next time

File: packages/kauri-components/components/Typography/Heading.re [hunk 13]
```diff
@@ -1,21 +1,25 @@
 module Styles = {
-  let heading =
+  let heading = (~colorProp, ~sizeProp) =>
     Css.(
       [%css
         {|
            {
-               font-size: 20px;
-               font-weight: 600;
+               font-size: px(sizeProp);
```
Comment (rej156) [inline packages/kauri-components/components/Typography/Heading.re]: Yeah we should just remove the ppx, it just looks silly tbh

File: packages/kauri-web/routes.js (hunk 5 above)
Comment (rej156) [reply packages/kauri-web/routes.js]: If you're omitting, please don't forget to change the this.props.url && this.props.url.query['collection'] in the respective pages/collection.js

File: packages/kauri-web/components/containers/Collection/View.js (hunk 6 above)
Comment (davodesign84) [reply packages/kauri-web/components/containers/Collection/View.js]: something like ``` if (!this.props.data || !this.props.data.collection) return null; ```

File: packages/kauri-web/components/containers/Collection/View.js (hunk 7 above)
Comment (davodesign84) [reply packages/kauri-web/components/containers/Collection/View.js]: ooops that was just a leftover

File: packages/kauri-components/components/Headers/CollectionHeader.re (hunk 10 above)
Comment (davodesign84) [reply packages/kauri-components/components/Headers/CollectionHeader.re]: Heading already comes with defaults :P 

``` let make = (~text, ~color="#333333", ~size=20, _children) => { ```

File: packages/kauri-web/queries/Collection.js (hunk 12 above)
Comment (davodesign84) [reply packages/kauri-web/queries/Collection.js]: I'm reluctant until we are sure what the plan is :)

Comment (rej156) [reply packages/kauri-web/queries/Collection.js]: Haha let's discuss this soon but yeah 

Queries done in ReasonApollo break when the schema is updated and we pull it down so the compiler shows which queries/components/containers/connections get broken
With clear line by line verbose error messages

Flow won't AS well is just my opinion/experience

**Pull request merged**